src/lib.rs

```diff
@@ -1,6 +1,8 @@
-#![deny(missing_docs)]
-//! Dummy crate needs high-level documentation.
-/// Dummy public function needs documentation.
-pub fn it_works() {
-    assert!(true);
-}
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//! Manages system resources that can be allocated to VMs and their devices.
+
+extern crate libc;
```

Comment:
https://doc.rust-lang.org/nightly/edition-guide/rust-2018/path-changes.html

due to rust 2018 edition guide, there is no need for extern crate. Please check the document to see if it's needed in your circumstance:)